### ISSUE TITLE
🐛 BUG: Using component from workspace with image fails starting with 0.18.x

### ISSUE THREAD

**Issue by radiosilence** [snowpackjs/astro#915]
### What package manager are you using?

yarn

### What operating system are you using?

macOS

### Describe the Bug

If I have a component imported from a workspace project, and that includes an image, it breaks with the error:

```
TypeError: Cannot read property 'code' of undefined
    at file:///xxx/my-astro-project/web/src/pages/index.astro
    at FileBuilder.getProxy (/xxx/my-astro-project/node_modules/snowpack/lib/cjs/build/file-builder.js:267:48)
    at Object.loadUrl (/xxx/my-astro-project/node_modules/snowpack/lib/cjs/commands/dev.js:628:55)
    at async load (/xxx/my-astro-project/node_modules/snowpack/lib/cjs/commands/dev.js:217:28)
    at async /xxx/my-astro-project/node_modules/snowpack/lib/cjs/ssr-loader/index.js:60:28
    at async /xxx/my-astro-project/node_modules/snowpack/lib/cjs/ssr-loader/index.js:83:23
    at async initializeModule (/xxx/my-astro-project/node_modules/snowpack/lib/cjs/ssr-loader/index.js:88:39)
    at async initializeModule (/xxx/my-astro-

```

If we log out the arguments to `async getProxy(_url, type) {` in file-builder this is what we get:

```
[build] ! building pages...
{
  _url: '/_snowpack/link/ui/src/images/twitter/twitter.png',
  type: '.js'
}
this.resolvedOutput {
  '.png': {
    code: <Buffer 89 50 4e 47 0d 0a 1a 0a 00 00 00 0d 49 48 44 52 00 00 00 1c 00 00 00 18 08 06 00 00 00 e9 9c 9d 82 00 00 00 09 70 48 59 73 00 00 0b 13 00 00 0b 13 01 ... 407 more bytes>,
    map: undefined
  }
}
```

### Steps to Reproduce

See repo

### Link to Minimal Reproducible Example (Optional)

https://github.com/radiosilence/astro-bug-915

**Comment by radiosilence** [comment]
Can anyone confirm that they can at least reproduce this with my repo?

**Comment by matthewp** [comment]
This is fixed with the latest snowpack release (3.8.6), updating your deps should resolve this now.
